### ISSUE TITLE
Notifications sometimes don't scroll to the specific comment in rants

### ISSUE THREAD

**Issue by Parou** [devRant/devRant#182]
I started to notice that right after the "Scroll to bottom" update.

Sometimes when I tap a notification in the android app like "XY mentioned you in a comment!" or "XY ++'d your comment!", the rant won't scroll down to the comment but stays at the top.

When this happens I go back to the notifications, do the same again and it works.

**Comment by DFoxinator** [comment]
Hmmm, that's really weird. Does it seem to be specific rants that cause it?

**Comment by 7twin** [comment]
Can confirm, I had the same thing happening randomly, but couldn't recreate it as consistent to open an issue.

**Comment by Parou** [comment]
No, it doesn't affect specific rants. It just sometimes doesn't work and if you go back and tap the same notification again, it works.

**Comment by Parou** [comment]
Just realized that it now happens with weekly rants all the time (when I opened this issue it was only sometimes and not related to wk rants but I think it's still related to this issue)
